### DISCUSSION TITLE
Couldn't resolve server from bootstrap.servers as DNS resolution failed

### DISCUSSION THREAD

**Discussion by AlexVegaIT** [tchiotludo/akhq#1264] [Q&A] [answered]
Hi,
I have a configuration which was working fine till today. Nothing has been changed, but I started receiving this error message today

"Couldn't resolve server *** from bootstrap.servers as DNS resolution failed for ***"

![image](https://user-images.githubusercontent.com/104902192/201949729-e6394ba6-266d-4312-b1f7-6e1013234714.png)

The same Kafka cluster host is working fine with other Kafka UI tool (Provectus).
The AKHQ instance I'm starting with docker-compose using yml configuration file.

How to resolve this issue and what is the root cause?

**Comment by AlexVegaIT** [comment, marked as answer]
An update from this morning, I've resolved this issue within next steps

- Run the AKHQ container with "docker-compose -f file_name.yml up"
- Open another WSL window (if you are using Windows OS) and type "docker container list" to get a container ID
- In that same terminal/wsl then type "docker exec -u root -it CONTAINER_ID /bin/bash"
- Go to /etc folder and type "echo "nameserver 1.1.1.1" > resolv.conf"

Right after resolv.conf is amended AKHQ app will start resolving listed clusters, topics etc.
I hope that this will trigger you to investigate this and someone to help resolve this temporary.

**Reply by tchiotludo** [reply]
Seems to be really related to your laptop configuration, no one I know have the issues since by default docker will use host dns.

**Comment by AlexVegaIT** [comment]
Or easier way would be to add DNS list in a composer file

  akhq:
    image: tchiotludo/akhq
    dns:
      - 1.1.1.1
      - 8.8.8.8
